app/serializers/story_serializer.rb

```diff
@@ -43,9 +43,9 @@ def key
     "Story_#{object.id}"
   end
 
-  # cached
-  #
-  # def cache_key
-  #   ['v5', object]
-  # end
+  cached
+
+  def cache_key
+    ['v3', object]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.